gx/gx-security-vaadin/src/main/java/io/graphenee/security/vaadin/GxUserAccountListPanel.java

```diff
@@ -82,6 +83,7 @@ protected String panelCaption() {
 
 	@Override
 	protected void onAddButtonClick(GxUserAccountBean entity) {
+		entity.setAccountActivationDate(new Timestamp(System.currentTimeMillis()));
```

Comment:
@farhanghouri 
Since AccountActivationDate is timestamp it should be set when user Saves the record because it is possible user clicks Add button but saves after 20minutes.